eng/versioning/external_dependencies.txt

```diff
@@ -38,7 +38,7 @@ io.netty:netty-codec-http2;4.1.51.Final
 io.netty:netty-handler;4.1.51.Final
 io.netty:netty-handler-proxy;4.1.51.Final
 io.netty:netty-tcnative;2.0.31.Final
-io.netty:netty-tcnative-boringssl-static;2.0.31.Final
+io.netty:netty-tcnative-boringssl-static;2.0.30.Final
```

Comment:
Why did we downgrade this from 2.0.31 to 2.0.30?

Reply:
We should not downgrade this, since this is being brought in as part of reactor-netty upgrade - 0.9.10.RELEASE, which brings in netty - 4.1.51.Final version and tcnative boring ssl version 2.0.31.Final

Reply:
Thank you for your check, I'll recover it.